src/main/java/exercise_1/TestCircle.java

```diff
@@ -0,0 +1,10 @@
+package exercise_1;
```

Comment:
never use underscore in package definition

Reply:
Yes sir.